### ISSUE TITLE
App crashes when adding a file to the request body with wrong permissions

### ISSUE THREAD

**Issue by Tilogorn** [a85/POSTMan-Chrome-Extension#10286]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the Issue

I `$ sudo cp /var/log/apache2/error.log ~/Desktop/` to send it as request body with postman. I forgot to adapt the permissions, Postman crashes a few moments after hitting "Send". Details see in "Steps To Reproduce".

### Steps To Reproduce

1. Open a new request tab, PUT to http://localhost
2. Select the Body tab
3. Select the `binary` type 
4. Open file explorer with "Select file"
5. Choose a file with these permissions: `-rw-r-----  1 root  root   284 Sep  9 13:12 error.log`
6. Click "Send"
7. App window just closes after 1-2 seconds

### Environment Information

```markdown
- Operating System: Ubuntu 20.04
- Platform Type: Native App
- Postman Version: 8.12.1
```

**Comment by timbochamp** [comment]
Hi @Tilogorn , thanks for reporting. I'm not able to reproduce this by making a small error.log file with a test string and copying it so its permissions match what you specified
![Screen Shot 2021-09-16 at 9 49 47 AM](https://user-images.githubusercontent.com/4521134/133615600-8d05113c-6e83-4b7e-987b-1a0b74fd5208.png)

Can you share any other details about the PUT request you're using if you're able to share it? I'm also curious the size of the error.log file on your end if you can share that too?

**Comment by Tilogorn** [comment]
Hi @timbochamp, not sure if you reproduced that correctly as I can see

1. Windows-like menu buttons in your Postman UI, but that may be due to your custom linux theming or network setup
2. That you have set `644` as file permissions, while I reported the problem with `640` 

If you're still not able to reproduce I'll share the same screenshots as you and/or make a gif of the crash. Your request seems correctly configured, although I just noticed that it doesn't make a difference if you're `GET`ing or `PUT`ting. My `error.log` contains just two lines of logging (size `284K`).
